examples/music-bot/tsconfig.json

```diff
@@ -2,7 +2,6 @@
 	"extends": "../../tsconfig.json",
 	"compilerOptions": {
 		"baseUrl": ".",
-		"outDir": "dist",
```

Comment:
Already included in the parent `tsconfig.json` https://github.com/discordjs/voice/blob/main/tsconfig.json#L29